src/utils/hashFeedName.ts

```diff
@@ -9,3 +9,7 @@ export function hashFeedName(feenName: FeedName): HexString {
 export function hashFeedName0x(feenName: FeedName): HexStringWith0x {
   return ethers.utils.id(feenName);
 }
+
+export function feedNameToBaseAndQuote(symbol: string): string[] {
+  return symbol.split('-');
```

Comment:
imo it is not worth creating separate method for it

Reply:
Sure, I can remove this method.

What I thought, when I created it, was that its advantage is that we don't have to hardcode on every place the split with the `-` character.
A hardcoded split might be a possible source of error in the future, difficult for it to occur, but possible in the end.
A developer, for example, might change the `-` for a `_` in one of the fetchers that is not being used at that moment and the error remains there for some months, and then when the fetcher is used this comes to light. Also, unit testing the method avoids that issue too.

The methods above are also pretty small, but they uniform the way the feedName is being threated.
For example, in all the cases using the `ethers` library.

Reply:
I rather want to have service that doing checks I mentioned, we can not do default `-` and forget about problem.

Reply:
done